bears/actions/AddNewlineAction.py

```diff
@@ -0,0 +1,28 @@
+from coalib.misc.Shell import run_shell_command
+from coalib.results.result_actions.ResultAction import ResultAction
+
+
+class AddNewlineAction(ResultAction):
+
+    SUCCESS_MESSAGE = 'New Line added successfully.'
+
+    def __init__(self, shortlog, body):
+        self.shortlog = shortlog
+        self.body = body
+
+    @staticmethod
+    def is_applicable(result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=()):
+        return 'AddNewlineAction' not in file_diff_dict
+
+    def apply(self, result, original_file_dict, file_diff_dict, **kwargs):
+        """
+        Add (N)ewline
+        """
+        new_commit_message = '{}\n\n{}'.format(self.shortlog, self.body)
+        command = 'git commit --amend -m "{}"'.format(new_commit_message)
+        stdout, err = run_shell_command(command)
+        file_diff_dict['AddNewlineAction'] = True
```

Comment:
Using `file_diff_dict` to know when this action is applied. This is required to know if action is applicable or not.